docs/client/operations.md

```diff
@@ -0,0 +1,177 @@
+# <img align="center" src="../images/logo.png">  Calling Operations with Your C# Client
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
+Depending on your swagger definition, operations can be accessed through operation groups (TODO: link to swagger docs) on the client,
+or directly on the client.
+
+## Operation Group vs No Operation Group
+
+If your swagger defines an operation group for your operation (for example, in [this][operation_group_example] swagger, the operation `list`
+is part of operation group `application`), you would access the operation through the client for that operation group, which in this case
+would be `ApplicationClient.List(...)`.
+
+If there's no operation group, as in [this][mixin_example] case, you would access the operation directly from the main client
+itself, i.e. `PetsClient.GetDog()`. Our responses are returned as responses of `T`, so if we want to access the value, we call `.Value`
+on the response object.
+
+## Regular Operations
+
+### Sync Operations
+
+We will be using the [example swagger][pets_swagger] in our main docs repo. After [initializing][initializing] our client, we
+call our operation like this:
+
+```csharp
+using Azure.Pets;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+var client = new PetsClient(endpoint);
+Dog dog = client.GetDog().Value;
+```
+
+### Async Operations
+
+We can also use our client to call async operations, and these operations are all suffixed with `Async`. Following the [example above](#sync-operations "Sync Operations"),
+our call to `GetDogAsync` looks like this:
+
+```csharp
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Pets;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+var client = new PetsClient(endpoint);
+Dog dog = (await client.GetDogAsync()).Value;
```

Comment:
Isn't there an implicit Operation<Dog> -> Dog conversion that does the .Value for us here?